### DISCUSSION TITLE
Panopticon messages - Site Up/ Site Down Messages suddenly being sent

### DISCUSSION THREAD

**Discussion by david-unwin** [akeeba/panopticon#602] [Q&A (Community Support)] [answered]
Only in the last week we have started getting messages from Panopticon about our site being down and then back up.
We get a message about our site being down and then a minute later, it being back up. (see attached screenshot)
We have been running Panopticon for a number of months now and never had these messages before.
Any reason, apart from the obvious( that the site goes down and comes back with a minute) why we are sudenly getting these messages.
It just seems strange that we have not had these messages before and now we get them once a day??
Kind regards
David
![webiste down](https://github.com/akeeba/panopticon/assets/95095238/5b4da7d5-7f84-4a5c-8d10-a23282567836)

**Comment by nikosdion** [comment, marked as answer]
You are getting these messages because the site goes down, and then comes back up. I assume these are the times of the day when the host installs updates and has to do a quick server restart.

As to why you didn't catch it before: if it's a short outage (a few seconds) it is very possible that the site goes down after we have just checked for uptime, and is back up before the next check for uptime. Think of it like this:
* 08:49:03 we check for uptime; site is up
* 08:49:24 the site goes down
* 08:49:54 the site goes back up
* 08:50:03 we check for uptime; site is up
In this case we are blind to the downtime.

Now, transpose the downtime start by 10 seconds:
* 08:49:03 we check for uptime; site is up
* 08:49:34 the site goes down
* 08:50:03 we check for uptime; site is down; email is sent
* 08:50:04 the site goes back up
* 08:51:03 we check for uptime; site is up; email is sent claiming downtime was 1 minute

So, nothing weird about it, it's how uptime monitoring works. Not just in Panopticon, but across all uptime monitoring services. You can't hit a server every second to see if it's up; you'd be DoS'ing yourself :)

**Comment by david-unwin** [comment]
Hi Nicholas,
Thanks for your detailed explanation, very much appreciated.
Kind regards,
David
